package.json

```diff
@@ -18,9 +18,13 @@
     "format": "prettier \"**/*.{js,json,md,ts,yml}\"",
     "format:check": "npm run format -- --check",
     "format:write": "npm run format -- --write",
-    "lint": "eslint \"src/**/*.ts\" --max-warnings 0",
+    "lint": "npm-run-all \"lint:*\"",
+    "lint:docs": "markdownlint \"**/*.md\"",
+    "lint:eslint-docs": "npm-run-all \"update:eslint-docs -- --check\"",
+    "lint:js": "eslint . --max-warnings 0 --report-unused-disable-directives",
     "prepublishOnly": "tsc",
-    "test": "jest --runInBand"
+    "test": "jest --runInBand",
+    "update:eslint-docs": "eslint-doc-generator"
```

Comment:
Suggest adding a note in `README.md` (or some other doc) about how to update the docs, run the tests, publish the package, etc.

Reply:
Agreed! https://github.com/JoshuaKGoldberg/eslint-plugin-expect-type/issues/70